hack/testing/util.sh

```diff
@@ -717,13 +717,15 @@ get_mux_pod_log() {
 get_rsyslog_pod_log() {
     local pod=${1:-$( get_running_pod rsyslog )}
     local logfile=${2:-/var/log/rsyslog/rsyslog.log}
-    oc logs $pod 2>&1
-    if oc exec $pod -- logs 2>&1 ; then
+    oc logs $pod -c rsyslog 2>&1
+    if oc exec $pod -c rsyslog -- logs 2>&1 ; then
         : # done
     elif oal_sudo test -f $logfile ; then
         # can't read from the pod directly - see if we can get the log file
         oal_sudo cat $logfile
     fi
+    oc logs $pod -c logrotate 2>&1
+    oc exec $pod -c logrotate -- logs 2>&1 || :
```

Comment:
does `logs` get both the /var/log/rsyslog/logrotate.log and /var/lib/rsyslog.pod/logrotate.log?

Reply:
Currently, it doesn't.  (and actually, "oc exec $pod -c logrotate -- logs" does not work as expected. It just reads /var/log/rsyslog/rsyslog.log...)  I'm going to fix rsyslog/utils/logs to cover these logrotate logs...